css/style.css

```diff
@@ -0,0 +1,65 @@
+@media only screen and (max-width: 1000px){
+
+}
+.welcome-section{
+    display: flex;
+    flex-direction: column;
+    align-items: center;
+    width: 100%;
+    height: 100vh;
+    justify-content:  center;
+    background-color: #002;
+}
+.nav{
+    display: flex;
+    justify-content: flex-end;
+    position: fixed;
+    background: crimson;
+    top: 0;
+    left: 0;
+    width: 100%;
+    z-index: 10;
+}
+p{
+    color: whitesmoke;
+    font-size: 2rem;
+}
+h1{
+    color: white;
+    font-size: 7rem;
+   
```

Comment:
- [ ] Kindly remove this extra line.

Reply:
Great review so far, It would be more efficient to assign HTML tags such as `<p>` and `<h1>` to CSS classes as this practice helps you write dry code in line with these specifications on [How to write dry code](https://time2hack.com/dry-frontend-html-css/)